apps/extension/package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "dev": "webpack --watch --mode=development -d inline-source-map",
     "clean": "rm -rfv dist bin",
-    "build": "webpack",
+    "prebuild": "download-keys",
+    "build": "KEY_PATH=$(download-keys) webpack",
```

Comment:
don't need both of these

Reply:
```
  "scripts": {
    "dev": "webpack --watch --mode=development -d inline-source-map",
    "clean": "rm -rfv dist bin",
    "build": "webpack",
    "lint": "eslint \"**/*.ts*\"",
    "test": "vitest run"
  }
```

why wouldn't we keep the build script `"build": "webpack"`?